lmfit/ui/ipy_fitter.py

```diff
@@ -116,12 +107,11 @@ def close(self):
         self.max_checkbox.close()
 
     def _repr_html_(self):
-        box = Box()
+        box = HBox()
         box.children = [self.value_text, self.vary_checkbox,
                         self.min_text, self.min_checkbox,
                         self.max_text, self.max_checkbox]
         display(box)
```

Comment:
When I test this in an py2 environment it works fine with the widgets as they are. This is what I thought would stop py2 compatibility, this is a quick test:
```
%matplotlib inline
import numpy as np
import lmfit
from lmfit.ui import Fitter
data = np.array([1,2,3,4,5,4,3,2])
x = np.arange(len(data))/2.0

Fitter( x=x, data = data)
```